### DISCUSSION TITLE
ED133UT3 Screen Suport + 13.3" alternatives

### DISCUSSION THREAD

**Discussion by dengfengdz** [vroland/epdiy#342] [General]
Hi ，guys！
I tried to use the ED133UT2 code to light up ED133UT3, but it didn't work properly and the screen didn't respond at all. I only have the ED133UT3 specification at present, and I didn't see any difference between them。

**Comment by dengfengdz** [comment]
[VB3300-NCC (ED133UT3).pdf](https://github.com/vroland/epdiy/files/13668042/VB3300-NCC.ED133UT3.pdf)

**Comment by martinberlin** [comment]
![VGH](https://github.com/vroland/epdiy/assets/2692928/62d51ec6-e543-4479-b9e2-3402edf21240)
Hi there @vroland if I see this correctly in the datasheet mentions Gate positive supply to be 27 V. If that is different from ED133UT2 it could be that the Gate of the pixel FETs is not receiving enough voltage to open and let pass the +15 volts right?
Can that be the issue @dengfengdz is experimenting here?

**Comment by vroland** [comment]
Hm, it could be. In practice it looked like +22V worked for all displays, although some specify a higher voltage. But it could be that this batch is particularly sensitive. The voltage can be raised by writing to some registers in the TPS65185, but this is not implemented in the public interface of epdiy. 
But it could also be that something is different with the pinout from UT2, I don't have a datasheet available on this machine, but it should be available somewhere to compare.

**Comment by martinberlin** [comment]
Hello @dengfengdz please provide feedback about this or close the issue. 
I'm not a big fan of keeping issues forever open since then we are lost trying to give support.

**Comment by jonardan** [comment]
Hello, i am using a ED133UT3 with an epdiy v5 and actually it works, but compared to a board from gooddisplay the colors are a bit washed out and don't look too good, I just bought an epdiy v7 to see if it will improve the image quality (because right now i'm setting vcom manually and i'm not too sure the value is exactly correct) do you think that issue with image quality is related to waveform file or it could be the vcom or something else?

**Reply by martinberlin** [reply]
Not sure. But yes try different waveforms, they are not resolution dependant. Try also the Lilygo one ED042TC2

**Comment by martinberlin** [comment]
@jonardan please send me an E-Mail to martin@cale.es and I will send you a new Waveform to try in the 13.3" display?

**Comment by martinberlin** [comment]
Planning to test also a 13.3" but is another model, different connector: https://github.com/vroland/epdiy/discussions/341
There you can find a poll to see if there is real interest of other developers too to try this model.
@jonardan can you please provide some photos of the rendered Dragon to see how it looks like ?

In AliExpress it seems there is also EL133US1 black white and red E-paper screen three-color label. It has same connector, but actually I've no idea of how the colors are driven, I suspect that there is just one pixel black, next red, datasheet for the exact model not found but this one seems to be a replacement model:
https://www.data-modul.com/sites/default/files/products/EA2220-NCC-specification-12051790.pdf

**Comment by jonardan** [comment]
Hi Martin, unfortunately i left the display in my office and am now in vacation, as soon as I come back I will post a photo

**Comment by milant42** [comment]
Hi @martinberlin nad @jonardan,

I'm trying out ED133UT3 with epdiy v7 (setting it as ED133UT2 in code), but the examples don't work - for example the dragon is not over the entire screen (but that might be an easy fix).

Did you manage to get ED133UT3 to work?

**Reply by martinberlin** [reply]
The dragon image is only 1200x825 for the 9.7” display so it will not be in the entire panel on the 13.3

try generating a bigger image with the python script or just try out the www-image examples, just update settings to match your WiFi and update display in the selected main file (there are 2 examples)

**Reply by milant42** [reply]
It was indeed the image size, successfully used image generator. Thank you!

**Comment by martinberlin** [comment]
There is no more 13.3 inches in Aliexpress. Only one I found is: ES133TT3  But I really don't know if I would try it. Not willing to make my displays collection bigger

**Comment by Konstantin-G** [comment]
Hi, I have a [GDEP133UT3](https://v4.cecdn.yun300.cn/100001_1909185148/GDEP133UT3.pdf) from Good Display,
tried it with your V7 and with their [Parallel Screen Test Board Development Kit, EPDiy ](https://www.good-display.com/product/510.html) . Both sims to not working how they should. Same [example](https://github.com/LaskaKit/Testcode_examples/tree/main/Displays/E-Paper/13-30/GDEP133UT3_epdiy/dragondemo) , Voltages the same: -15V , +15V, -20V, +22V and VCOM= -2,23V   
Good Display board: Get full image but no details. 
<img width="1252" alt="image" src="https://github.com/user-attachments/assets/4f4ea535-a641-486d-935d-10e35bc2db63" />
V7: Details looks OK, but too bright (also tried change +22V to +27V on the board, same result):
<img width="1391" alt="image" src="https://github.com/user-attachments/assets/20d92564-7eeb-46e6-b7db-37cc26afd2de" />
Can you guess where is the problem?
Thanks a lot.

**Reply by martinberlin** [reply]
It's the grays matrix also called Waveform that is not ideal. Please try to switch to ED042TC2 but then it will have more grays and will be a bit darker as it should. Send me a private E-Mail I will try forwarding you a waveform that is not public.
Also I invite you to read my [blog post about controlling Einks](https://fasani.de/2025/01/03/on-driving-eink-displays/)
